### DISCUSSION TITLE
Refer to the code of multiviewport example but it doesn't work

### DISCUSSION THREAD

**Discussion by iamazy** [emilk/egui#5440] [Q&A] [answered]
My code as follows:
![image](https://github.com/user-attachments/assets/e3f0a323-b65d-458c-aa39-2e7f859c1aa4)

Could you please give me some suggestions?

**Comment by lucasmerlin** [comment, marked as answer]
Looking at [the docs](https://docs.rs/egui/latest/egui/struct.Context.html#method.show_viewport_deferred), you need to call show_viewport_deferred each frame, not only once when the button is clicked.
